es_ES/flutter-for-ios.md

```diff
@@ -0,0 +1,1937 @@
+* `decorationStyle`
+* `fontFamily`
+* `fontSize`
+* `fontStyle`
+* `fontWeight`
+* `hashCode`
+* `height`
+* `inherit`
+* `letterSpacing`
+* `textBaseline`
+* `wordSpacing`
+
+# Entrada de formulario
+
+## ¿Cómo funcionan los formularios en Flutter? ¿Cómo puedo recuperar los datos introducidos por el usuario?
+
+Dada la forma en que Flutter utiliza widgets inmutables con un estado separado, puede que te preguntes cómo encaja la entrada del usuario en la imagen. En iOS, normalmente consultas los widgets por sus valores actuales cuando llega el momento de enviar la entrada del usuario, o una acción sobre ella. ¿Cómo funciona eso en Flutter?
+
+En la práctica, los formularios son manejados, como todo en Flutter, por widgets especializados. Si usted tiene un `TextField` o un `TextFormField`, se puede asignar un
+[`TextEditingController`](https://docs.flutter.io/flutter/widgets/TextEditingController-class.html)
+para recuperar las entradas del usuario:
+
+<!-- skip -->
+{% prettify dart %}
+class _MyFormState extends State<MyForm> {
+  // Crea un controlador de texto y utilícelo para recuperar el valor actual.
+  // del campo de texto!
+  final myController = TextEditingController();
+
+  @override
+  void dispose() {
+    // Limpia el controlador cuando elimine el Widget.
+    myController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Recuperar entrada de texto'),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: TextField(
+          controller: myController,
+        ),
+      ),
+      floatingActionButton: FloatingActionButton(
+        // Cuando el usuario pulsa el botón, muestra un diálogo de alerta con el
+        // texto que el usuario ha escrito en    nuestro campo de texto.
+        onPressed: () {
+          return showDialog(
+            context: context,
+            builder: (context) {
+              return AlertDialog(
+                // Recupera el texto que el usuario ha tecleado con nuestro
+                // TextEditingController
+                content: Text(myController.text),
+              );
+            },
+          );
+        },
+        tooltip: 'Show me the value!',
+        child: Icon(Icons.text_fields),
+      ),
+    );
+  }
+}
+{% endprettify %}
+
+Puede encontrar más información y la lista completa de códigos en
+[Recuperar el valor de un campo de texto](/cookbook/forms/retrieve-input/),
+desde el [Flutter Cookbook](https://flutter.io/cookbook/).
+
+## ¿Cuál es el equivalente de un placeholder en un campo de texto?
+
+En Flutter puede mostrar fácilmente una "sugerencia" o un texto de referencia para su campo añadiendo un objeto`InputDecoration` al parámetro constructor de decoración para
+los `Text` widget:
+
+<!-- skip -->
+{% prettify dart %}
+body: Center(
+  child: TextField(
+    decoration: InputDecoration(hintText: "Esto es una pista"),
```

Comment:
This is a hint
Como criterio general solo traducimos los comentarios en el código